data/remotes.d/automatic-vendor.conf

```diff
@@ -0,0 +1,7 @@
+[fwupd Remote]
+# this configuration provides dynamically generated metadata shipped by the OS vendor and can
+# be found in @datadir@/fwupd/remotes.d/vendor/firmware
+Enabled=false
+Title=Vendor (Automatic)
```

Comment:
I think a new .conf file makes sense, but I'm not sure about the `automatic-vendor.conf` name. Maybe `vendor-directory.conf` or something?

Reply:
OK, renamed to `vendor-directory.conf`